tests/unit/components/shared/BankData.spec.ts

```diff
@@ -28,7 +28,7 @@ describe( 'BankData', () => {
 			},
 			store: createStore(),
 			mocks: {
-				$t: () => {},
+				$t: ( key: string ) => key,
```

Comment:
You could define a function `fakeTranslate` at the top and reference it everywhere. That way we can have the type and the definition in one place and not repeat it so much